wechatpy/pay/__init__.py

```diff
@@ -28,7 +28,8 @@ class WeChatPay(object):
     微信支付接口
 
     :param appid: 微信公众号 appid
-    :param api_key: 商户 key
+    :param sub_appid: 当前调起支付的小程序APPID
+    :param api_key: 商户 key,不要在这里使用小程序的密钥
```

Comment:
We were very confused on trying to make payment via mini-program work. The idea is that mini-program's api key is not used to make pay requests, while it's used in [sns/jscode2session](https://developers.weixin.qq.com/miniprogram/dev/api/api-login.html)